lxc/remote.go

```diff
@@ -326,6 +326,18 @@ func (c *cmdRemoteAdd) Run(cmd *cobra.Command, args []string) error {
 		}
 
 		conf.Remotes[server] = config.Remote{Addr: addr, Public: true, Protocol: c.flagProtocol}
+
+		// Check that the server is reachable before saving the config
+		d, err := conf.GetImageServer(server)
+		if err != nil {
+			return err
+		}
+
+		_, err = d.GetImages()
```

Comment:
Does that get a list of images? If yes, a `HEAD` request might accomplish the same but more quickly?

Reply:
I just tested it and it's probably not that big of a deal:
```
$ time lxc image list ubuntu: >/dev/null

real	0m3.811s
user	0m1.398s
sys	0m0.226s
```